.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
 				allowedTextDomain: 'sensei-lms',
 			},
 		],
+		'react-hooks/exhaustive-deps': 'warn'
```

Comment:
@yscik I noticed that this rule wasn't active. I checked that the `wp-scripts` includes the `rules-of-hooks` only. So I added it because it prevents some mistakes while using some hooks. ;)

I added directly inner this PR instead of opening a new because I already fixed some rules in this PR.